kubernetes/resource_kubernetes_mutating_webhook_configuration.go

```diff
@@ -109,7 +109,7 @@ func resourceKubernetesMutatingWebhookConfiguration() *schema.Resource {
 						"rule": {
 							Type:        schema.TypeList,
 							Description: webhookDoc["rules"],
-							Required:    true,
+							Optional:    true,
 							MinItems:    1,
```

Comment:
I think you can remove this as well, as if it is optional then the MinItems will be `0`.

Reply:
That makes sense, thank you, @jrhouston! Fixed.